services/inference/tests/test_inference_lib.py

```diff
@@ -94,6 +96,7 @@ def test_forecast_with_good_data(ts_data_base: pd.DataFrame, forecasting_input_b
     _basic_result_checks(results, df)
 
 
+@pytest.mark.parametrize("forecasting_input_base", [TTM_MODEL_ID], indirect=True)
```

Comment:
Do we need to parameterize if there is only one parameter? Or can we also add chronos here?

Reply:
assert for chronos is different. The resulting dataframe does not have date column. So, chronos should be in a separate test function. If any preprocessing and post processing of data is enabled for chronos, we will be able to use the same test.

Reply:
We need the result to contain a date column

Reply:
Chronos vs. TTM should produce a similarly formatted result, and take all the same inputs.

Reply:
Added date column in the output. Updated the chronos handler to use TimeSeriesPreprocessor to handle issues around empty target column list, frequency etc. This allowed to extend the TTM units to Chronos.